pyproject.toml

```diff
@@ -9,7 +9,7 @@ pythonpath = [
 ]
 
 [tool.black]
-line-length = 120
+line-length = 80
```

Comment:
could probably remove this section – i assume this is the default anyway

Reply:
>  Black defaults to 88 characters per line,

Reply:
that's so weird